### PR TITLE
Added side bar component

### PR DESCRIPTION
**Pull request by sanketbansal**
#### Changes proposed in this pull request:

- Added side bar as utility component for the side bar in the Dashboard UI.

- Link to live demo: http://pr-159-evalai.surge.sh 

Side-bar

### CODE REVIEW

**Comment by lunayach** [comment]
@sanketbansal Could you please attach the screenshot?

**Comment by sanketbansal** [comment]
@lunayach @Shekharrajak @RishabhJain2018 @galipremsagar  please review

**Comment by lunayach** [comment]
@sanketbansal this is not reflected in the UI right? That is, I don't see the selector being called nor it's added to the routing module.

**Comment by sanketbansal** [comment]
It is the utility component used by other PRs

File: src/app/app.module.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Import Modules
+
```
Comment (lunayach) [inline src/app/app.module.ts]: Remove this empty line.

Comment (sanketbansal) [reply src/app/app.module.ts]: Done

File: src/app/components/utility/side-bar/side-bar.component.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      width: 100%;
+      height: 100%;
+      font-size: 14px;
+      transition: all 0.2s ease-in-out;
+      border-right: 1px solid transparent;
+      &:hover {
+        color: $highlight;
+      }
+      &.side-active-link {
+        /*background-color: $dust-gray;*/
```
Comment (lunayach) [inline src/app/components/utility/side-bar/side-bar.component.scss]: Remove this.

File: src/app/components/utility/side-bar/side-bar.component.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  z-index: 10;
+  top: 98px;
+  text-align: center;
+}
+
+.side-logo {
+  font-size: 20px;
+}
+
+.side-intro {
```
Comment (lunayach) [inline src/app/components/utility/side-bar/side-bar.component.scss]: If not needed, remove this class.

**Review by lunayach — APPROVED** [review]
@RishabhJain2018 we can merge this, I think. However, the UI changes would be visible in #141.

**Pull request merged**